### PR TITLE
Add: codes from the admission trail process into the new branch

### PR DESCRIPTION
**Pull request by mmsesay**
I added the following to this project

- Codes from the previous admission trial
- Setup linters configuration
- Fixed all linters errors

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+#headline {
+  margin: 50px 0;
+  text-align: center;
+}
+
+ul {
+  max-width: 550px;
+  margin: 0 auto 50px auto;
+  text-align: left;
```
Comment (mmsesay) [inline css/style.css:82]: Please try to use the CSS DRY method. You are repeating this line of code in more than one block.
Follow this link to get some understanding of the CSS DRY method: https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+li {
+  margin: 16px 0;
+}
+
+blockquote {
+  font-style: italic;
+  max-width: 545px;
+  margin: 0 auto 50px auto;
+  text-align: left;
```
Comment (mmsesay) [inline css/style.css:94]: Apply the CSS DRY rule to stop repeat these lines of code

**Review by mmsesay — COMMENTED** [review]
#Changes required:recycle: 

Please check my comments below and request a review when done.
Thank you. :smile:

**Review by Dipeshtwis — APPROVED** [review]
# Hi @mmsesay, 

### Status: Approved ✔️

:+1: The student used the `comment` feature from Github.
:+1: The student wrote at least 2 comments in the review.
:+1: The student always includes an explanation, a "why", for each requested change.

Your project is complete!  There is nothing to say, other than ... it's time to merge it. :detective: 
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**